src/main/java/net/atos/zac/app/inboxdocumenten/converter/RESTInboxDocumentConverter.java

```diff
@@ -1,23 +1,35 @@
 /*
- * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-FileCopyrightText: 2022 Atos, 2025 Lifely
  * SPDX-License-Identifier: EUPL-1.2+
  */
-
 package net.atos.zac.app.inboxdocumenten.converter;
 
+import static nl.info.client.zgw.util.UriUtilsKt.extractUuid;
+
 import java.util.List;
 import java.util.stream.Collectors;
 
+import jakarta.inject.Inject;
+
+import net.atos.client.zgw.drc.DrcClientService;
 import net.atos.zac.app.inboxdocumenten.model.RESTInboxDocument;
 import net.atos.zac.documenten.model.InboxDocument;
 
 public class RESTInboxDocumentConverter {
 
+    @Inject
+    private DrcClientService drcClientService;
+
     public RESTInboxDocument convert(final InboxDocument document) {
         final RESTInboxDocument restDocument = new RESTInboxDocument();
         restDocument.id = document.getId();
         restDocument.enkelvoudiginformatieobjectUUID = document.getEnkelvoudiginformatieobjectUUID();
         restDocument.enkelvoudiginformatieobjectID = document.getEnkelvoudiginformatieobjectID();
+        restDocument.informatieobjectTypeUUID = extractUuid(
+                drcClientService
```

Comment:
Is this really needed? I do my best to remove as many dependencies from those converter classes as I can because I think converters should be static / functional wherever possible. :-) 

With dependencies like this I would argue that these things are no longer proper converters. So my question is: can this logic not be pulled out of this converter so that this converter can be made static (which it should have been already)?

Reply:
Extracted DRCClientService usage out of converter